packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.types.ts

```diff
@@ -0,0 +1,55 @@
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { TextProps } from '@fluentui-react-native/text';
+import { IForegroundColorTokens, FontTokens } from '@fluentui-react-native/tokens';
+import { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import { RadioGroupContextValue } from './radioGroupContext';
+
+export const radioGroupName = 'RadioGroup';
+
+export interface RadioGroupState {
+  context: RadioGroupContextValue;
+}
```

Comment:
The pattern for the newer components has generally been to have the context value type inherit from the state type. This is because a lot of the stuff that goes on the context is based on internal state of the group component.

Reply:
Makes sense! Revised implementation to follow new component pattern for context provider.